IFC4x3/Properties/r/RadialExcavationDirection_0PWybQDsb7VuLXarptQq0N/DocProperty.xml

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Still having some issues with the constants. I assume `VERTICAL` means in the direction of gravity, right? I would celebrate some documentation.

Reply:
This comment should be answered by @JontyDunn